JitsiConference.js

```diff
@@ -1377,6 +1382,37 @@ JitsiConference.prototype.onIncomingCall
 };
 
 /**
+ * Sets the BridgeChannel.
+ */
+JitsiConference.prototype._setBridgeChannel = function() {
+    const jingleSession = this.jvbJingleSession;
```

Comment:
No, I mean that `this.jvbJingleSession` could be null if this function is called while in p2p mode if the jvb session is not fully established yet. Can you please add a check?